static/app/views/issueList/actions/index.tsx

```diff
@@ -400,8 +401,8 @@ function IssueListActions({
         onSelectStatsPeriod={onSelectStatsPeriod}
       />
       {!allResultsVisible && pageSelected && (
-        <StyledAlert type="warning" system>
-          <SelectAllNotice data-test-id="issue-list-select-all-notice">
+        <Alert system type="warning">
```

Comment:
hmm, the system usage feels wrong here, shouldn't we just be using a panelAlert here?

Reply:
I dont think we want the icon in this case, as it'll show the Warning Icon. This entire thing is more of a callout than an actual alert. It probably would have been better to have a pattern next to the checkbox for this. A dropdown with bulk actions might be preferable tbqh

Reply:
yeah this whole thing just sort of feels semantically wrong